conda_env/env.py

```diff
@@ -58,11 +72,75 @@ def from_yaml(yamlstr, **kwargs):
     return Environment(**data)
 
 
+def get_all_pip_dependencies(requirements_path, temp_dir='_delete_when_done'):
+    """
+    Use pip to gather all of the packages that would be installed for the given
+    requirements.txt file.
+    """
+    pip_cmd = ('pip', 'install', '--src', temp_dir, '--download', temp_dir,
+               '--no-use-wheel', '-r', requirements_path)
+    try:
```

Comment:
See comment elsewhere.  Please use isdir to check if dir exists, then create if necessary rather than try/except.